numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -275,7 +275,7 @@ def scrubF2CSource(c_file):
 def ensure_executable(name):
     try:
         which(name)
-    except:
+    except Exception:
         raise SystemExit(name + ' not found')
```

Comment:
```suggestion
    except Exception as ex:
        raise SystemExit(name + ' not found') from ex
```
Explicitly raising from the previous exception would be good here, especially since it can be an arbitrary exception.

Reply:
Thanks for the correction!